src/Matcher/ClassMethodCallReferenceResolver.php

```diff
@@ -40,12 +40,12 @@ public function resolve(MethodCall|MethodCallableNode $methodCallOrMethodCallabl
             return null;
         }
 
-        if (! $callerType instanceof TypeWithClassName) {
+        if (count($callerType->getObjectClassNames()) !== 1) {
```

Comment:
See https://phpstan.org/blog/why-is-instanceof-type-wrong-and-getting-deprecated